environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - scipy
   - pandas
   - netCDF4
+  - pygrib
```

Comment:
pygrib is actually not needed in pytesmo, but without it, the ``ascat`` package produces some import warnings